tests/lib/rules/no-implicit-injections.js

```diff
@@ -0,0 +1,288 @@
+//------------------------------------------------------------------------------
```

Comment:
Same here - can you add some test cases with multiple classes in one file with valid/invalid cases? And have you tested this on a codebase?

Reply:
I've tested this on our codebase of over 300 components and 200+ routes and it was able to identify and autofix all instances (65 usecases). I also checked with configuration below:

```js
{
  
  rules: {
    'ember/no-implicit-injections': [
      2,
      {
        services: [
          // Ember Responsive Used to Auto Inject the media service in Components/Controllers
          { serviceName: 'media', moduleNames: ['Component', 'Controller'] },
          // Ember CLI Flash Used to Auto Inject the flashMessages service in all modules
          { serviceName: 'flashMessages' },
          { serviceName: 'store', moduleNames: ['Route', 'Controller'] },
        ],
      },
    ],
  }
}```